spec/models/manageiq/providers/redhat/infra_manager_spec.rb

```diff
@@ -78,4 +78,128 @@
       expect(described_class.extract_ems_ref_id("/ovirt-engine/api/vms/123")).to eq("123")
     end
   end
+
+  context "api versions" do
+    require 'ovirtsdk4'
+    let(:ems) { FactoryGirl.create(:ems_redhat_with_authentication) }
+    subject(:supported_api_versions) { ems.supported_api_versions }
+    context "#supported_api_versions" do
+      before(:each) do
+        Rails.cache.delete(ems.cache_key)
+        Rails.cache.write(ems.cache_key, cached_api_versions)
+      end
+
+      context "when no cached supported_api_versions" do
+        let(:cached_api_versions) { nil }
+        it 'calls the OvirtSDK4::Probe.probe' do
+          allow(OvirtSDK4::Probe).to receive(:probe).and_return([])
+          expect(OvirtSDK4::Probe).to receive(:probe)
```

Comment:
Redundant, can combine to `expect(OvirtSDK4::Probe).to receive(:probe).and_return([])`